### DISCUSSION TITLE
Hasura Cloud Env Vars referencing each other

### DISCUSSION THREAD

**Discussion by TJackwood-Jasper** [hasura/graphql-engine#8373] [General]
We have a number of Hasura triggers set up which reference environment variabless for which API endpoints they should be triggering.

Ideally, I would like to have one environment variable set for our "Base URL" for our API, then have all of our other environment variables reference that base URL with an additional path for each endpoint, so that swapping to a different base URL is as easy as changing the single environment variable instead of updating every variable individually.

A quick example of what I'm looking for:

Env Vars:
 - MY_API_BASEURL -> https://myFunApi.com
 - HASURA_GRAPHQL_AUTH_HOOK -> ${MY_API_BASEURL}/api/auth
 - CUSTOM_EVENT_TRIGGER_URL_A -> ${MY_API_BASEURL}/api/triggers/doSomething
 - CUSTOM_EVENT_TRIGGER_URL_B -> ${MY_API_BASEURL}/api/triggers/doSomethingElse
 - etc. etc.

**Comment by tirumaraiselvan** [comment]
You can do this! Just set your webhook to be `{{MY_API_BASEURL}}/something`. Note that this should be set as static url and  not "from env".
